src/dynamichmc_inference.jl

```diff
@@ -33,7 +33,7 @@ function dynamichmc_inference(prob::DiffEqBase.DEProblem, alg, t, data, priors,
                               kwargs...)
     likelihood = sol -> sum( sum(logpdf.(Normal(0.0, σ), sol(t) .- data[:, i]))
                              for (i, t) in enumerate(t) )
-
+    println(typeof(transformations))
```

Comment:
Not sure why you are printing it, but that's a minor thing.

Regarding the API: perhaps I am missing something, but I would make transformations part of the problem, ie in `DynamicHMCPosterior`.

Reply:
You could also make the transformation a slot in the composite type. But with this interface, it may not be needed.

Reply:
Oh I see, the does sound good, but then how would the applied is not clear to me, I can imagine that with the `ContinuousTransformations` interface, I'll push with what I think would work here but I doubt it will be correct 😅